### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by ucasyjz** [magic-research/piecewise-rectified-flow#15]
I also have a question, the open-source SD1.5 weight is pre-trained with eps loss, can I directly use the weight for fine-tune with the rectified flow loss, is there will be a problem, can you give me some help, thank you!
